install_config/monitoring/configuring-etcd-monitoring.adoc

```diff
@@ -31,64 +31,16 @@ prometheus-k8s-1                               0/4       ContainerCreating   0
 prometheus-operator-6c9fddd47f-9jfgk           1/1       Running             0          36m
 ----
 
-. Open the configuration file for the cluster monitoring stack:
+. Set this variable to `true` in the Ansible inventory file:
 +
-----
-$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
-----
-
-. Under `config.yaml: |+`, add the `etcd` section.
-
-.. If you run `etcd` in static pods on your master nodes, you can specify the `etcd` nodes using the selector:
-+
-----
-...
-data:
-  config.yaml: |+
-    ...
-    etcd:
-      targets:
-        selector:
-          openshift.io/component: etcd
-          openshift.io/control-plane: "true"
-----
+`openshift_cluster_monitoring_operator_etcd_enabled`
```

Comment:
if we only set openshift_cluster_monitoring_operator_etcd_enabled=true without setting the kube-etcd-client-certs secret first, we would get error, the prometheus pod will not become running
`# oc -n openshift-monitoring get pod | grep prometheus-k8s
prometheus-k8s-0                              0/4       ContainerCreating   0          8m
`
`# oc -n openshift-monitoring describe pod prometheus-k8s-0
Events:
  Type     Reason       Age              From                        Message
  ----     ------       ----             ----                        -------
  Normal   Scheduled    1m               default-scheduler           Successfully assigned openshift-monitoring/prometheus-k8s-0 to juzhao-311-node-1
  Warning  FailedMount  2s (x8 over 1m)  kubelet, juzhao-311-node-1  MountVolume.SetUp failed for volume "secret-kube-etcd-client-certs" : secrets "kube-etcd-client-certs" not found
`
since cluster_monitoring_operator is installed by default, and ectd monitoring is not enabled by default, we don't need to change the enable monitoring part. All we need to do is just explain how to keep the ectd monitoring configuation after the OCP is upgrade. steps are
1. enable ectd monitoring
2. add openshift_cluster_monitoring_operator_etcd_enabled, openshift_cluster_monitoring_operator_etcd_hosts(If you run `etcd` on separate host) to inventory file
3. upgrade OCP
and the etcd monitoring configuration is kept after upgrade

Reply:
@juzhao not sure to follow your comment. The procedure described here is to setup the etcd monitoring and setting up the `kube-etcd-client-certs` secret is explained below.
As you wrote, it is assumed that etcd monitoring stays enabled after 3.11.x upgrades but IMO, it doesn't need to be documented.

Reply:
@simonpasquier this is the doc,
https://github.com/openshift/openshift-docs/blob/9bbfadf1d284416b0a21a524e28461ef183e1c75/install_config/monitoring/configuring-etcd-monitoring.adoc
step 2 and step 3, what should we do if we added the two ansible parameters?
followed the steps, we can not get to step 7